src/main/kotlin/uk/gov/justice/digital/hmpps/makerecalldecisionapi/service/RecommendationService.kt

```diff
@@ -67,8 +65,8 @@ internal class RecommendationService(
       status = updatedRecommendationEntity.data.status,
       recallType = updateRecommendationRequest.recallType
         ?: RecallType(
-          value = updatedRecommendationEntity.data.recommendation ?: existingRecommendationEntity.data.recommendation,
-          options = useExistingOptions()
+          value = updatedRecommendationEntity.data.recallType?.value ?: existingRecommendationEntity.data.recallType?.value,
```

Comment:
Can you not just use `recallType = updateRecommendationRequest.recallType` here?

Reply:
https://media.giphy.com/media/xT0xeJpnrWC4XWblEk/giphy-downsized-large.gif

Reply:
I don't think it would work :thinking_face: Shouldn't the response reflect the state of the updated resource? If the request had no recallType, and the request's recallType was used in the response. Then it would look like the resource in the DB had no recallType, when it fact it might have.